bot.py

```diff
@@ -45,8 +45,8 @@ async def help(ctx):
 async def prefix(ctx, value):
     """ Allows server administrators to change the bot prefix """
     if ctx.message.author.server_permissions.administrator:
-        os.environ['discord_bot_prefix'] = value
-        helpstr = os.environ['discord_bot_prefix'] + 'help'
+        os.environ['discord_prefix'] = value
+        helpstr = os.environ['discord_prefix'] + 'help'
```

Comment:
We may want to try an f string here: https://realpython.com/python-f-strings/

```suggestion
        helpstr = f"{os.environ['discord_prefix']}help"
```

Reply:
This looks great!